bridges/evm/contracts/StoreProofLib.sol

```diff
@@ -2,9 +2,15 @@ pragma solidity 0.5.14;
 
 import { BytesLib } from "./BytesLib.sol";
 
+/**
+ * @dev Helper library for calculating Merkle proof
+ */
 library StoreProofLib {
   using BytesLib for uint256;
 
+  /**
+   * @dev A group of data that necessary for computing appHash
```

Comment:
Explain what `prefixes` arrays, `path` arrays (should be renamed to `paths`?),  and value are.